res/sections/42-ComandiBase.tex

```diff
@@ -0,0 +1,109 @@
+
+\subsubsection{Buone regole per gli identificativi}
+Solitamente, agli identificativi utilizzati dalle \verb!label! e \verb!ref! vengono anteposti alcuni caratteri che permettono di identificare che tipo di oggetto è referenziato. 
+\begin{table}[]
+\centering
+\begin{tabular}{ll}
+\hline
+\textbf{prefisso} & \textbf{tipologia}         \\ \hline
+sec:              & section                    \\ \hline
+subsec:           & subsection                 \\ \hline
+fig:              & figure                     \\ \hline
+tab:              & tabella                    \\ \hline
+eq:               & formula                    \\ \hline
+lst:              & codice                     \\ \hline
+itm:              & elementi di lista numerata \\ \hline
+alg:              & algoritmo                  \\ \hline
+app:              & appendice                  \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{Collegamenti al Web}
+Il pacchetto che permette l'inserimento in maniera corretta in un documento \LaTeX{} di \textit{link} o \textit{URL} è \verb!hyperref!. Ci sono due modalità principali per inserire queste informazioni. La prima è \verb!\url{indirizzo}!, con la quale verrà mostrato l'URL in formato \textit{monospace}, pensato anche per documenti che devono essere stampati, la seconda è \verb!\href{indirizzo}{testo_da_mostrare}!, la quale non mostrerà esplicitamente l'indirizzo, ma mostrerà solamente il testo voluto (come succede nei siti Web per capirci). 
+
+\subsubsection{Email}
+Per inserire gli indirizzi email nel testo è preferibile creare questo comando personalizzato, da inserire nel preambolo del documento \verb!\newcommand{\mail}[1]{\href{mailto:#1}{\texttt{#1}}}! (per il momento fidatevi, spiegheremo in seguito come creare i propri comandi). L'utilizzo è molto semplice, infatti nel documento è sufficiente dare il comando \verb!\email{...}! specificando l'indirizzo di posta elettronica voluto.
+
+\subsubsection{Personalizzazione dei link}
+È buona norma, almeno nella creazione dei siti Web, colorare i link con un colore differente dal testo normale, in modo da evidenziare a chi visita la pagina che lì ci si può cliccare. Anche nei documenti non fa male evidenziare questi indirizzi e il pacchetto \verb!hyperref! permette la personalizzazione tramite il comando \verb!\hypersetup{...}! da mettere nel preambolo del documento, al quale devono essere specificate le personalizzazioni volute. Le più importanti sono:
+\begin{itemize}
+    \item \textbf{colorlinks=false} oppure \textbf{colorlinks=true}, che indica rispettivamente se si vuole che i link vengano riquadrati oppure colorati;
+    \item \textbf{urlcolor=color} che serve per specificare il colore dei link (sostituite \textit{color} con un colore in lingua inglese);
+\end{itemize}
+\href{https://en.wikibooks.org/wiki/LaTeX/Hyperlinks}{Per maggiori info sulla personalizzazione}.
```

Comment:
Io scriverei `Maggiori informazioni possono essere reperite nel [capitolo apposito](https://en.wikibooks.org/wiki/LaTeX/Hyperlinks) su WikiBook.` o metterei il link in chiaro che se qualcuno lo stampa poi si perde il link 😅  ma questo è perché non piace a me scritto così, vedi te poi cosa fare 😁